systers_portal/templates/meetup/snippets/user-cell.html

```diff
@@ -0,0 +1,15 @@
+{% load staticfiles %}
+
+{% if user.profile_picture.name and user.profile_picture.name != "False" %}
+  <a href="{{ user.get_absolute_url }}">
+    <img src="{{ user.profile_picture_thumbnail.url }}" class="profile-pic img-square img-responsive crop_img" alt="{{ user }} profile picture"/>
+  </a>
+{% else %}
+  <a href="{{ user.get_absolute_url }}">      
```

Comment:
So a suggestion: put the user profile thumbnail and name into a div, give it a set width, and then you can use `text-align: center;` and not indent anything.